cypress/e2e/tables-table.cy.js

```diff
@@ -36,15 +36,16 @@ describe('Manage a table', () => {
 		cy.uploadFile('test-import.csv', 'text/csv')
 		cy.contains('.app-menu-entry--label', 'Tables').click()
 		cy.contains('button', 'Create new table').click()
-		cy.get('.tile').contains('Import').click({ force: true })
+		cy.get('.tile').contains('Import table').click({ force: true })
 		cy.get('.modal__content').should('be.visible')
 		cy.get('.modal__content input[type="text"]').clear().type('import list')
 		cy.contains('button', 'Create table').scrollIntoView().click()
-		cy.contains('h2', 'Import').should('be.visible')
+		cy.contains('h2', 'Import table').should('be.visible')
 
 		cy.get('.modal__content button').contains('Select from Files').click()
 		cy.get('.file-picker__files').contains('test-import').click()
 		cy.get('.file-picker button span').contains('Choose test-import.csv').click()
+		cy.get('.modal__content button').contains('Preview').click()
 		cy.get('.modal__content button').contains('Import').click()
 		cy.get('[data-cy="importResultColumnsFound"]').should('contain.text', '4')
 		cy.get('[data-cy="importResultColumnsMatch"]').should('contain.text', '0')
```

Comment:
This test fails because when the `import-test.csv` file is imported, the matched columns is 4 not 0. Tested manually by importing the file myself and it does indeed say 4 matched columns